index.js

```diff
@@ -1,17 +1,51 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a,b){
+      return a - b
+    });
+    this.length = this.items.length;   
+  }
+  
+  get(pos) {
+    if (pos > this.items.length){
+     throw new Error('OutOfBounds')
+    }
+    return this.items[pos]    
+  }
 
-  get(pos) {}
+  max() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList')
+     }
+     return this.items[this.length -1]
+  }
 
-  max() {}
+  min() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList')
+     }
+    return this.items[0]
+  }
 
-  min() {}
+  sum() {
+    if (this.items.length === 0){
+      return 0
+     }
+    return this.items.reduce((acc,number) => acc + number)
+  }
 
-  sum() {}
-
-  avg() {}
+  avg() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList')
+     }
+    return this.sum()/this.items.length;
```

Comment:
Geralmente colocamos um espaço entre os dados e os operadores

`this.sum() / this.items.length`